src/librustdoc/html/static/main.js

```diff
@@ -385,6 +386,127 @@
                 }
             }
 
+            function transformResults(results, isType) {
+                var out = [];
+                for (i = 0; i < results.length; ++i) {
+                    if (results[i].id > -1) {
+                        var added = false;
+                        var obj = searchIndex[results[i].id];
```

Comment:
What kind of obj is this?

Reply:
The same as anywhere else in the code using `searchIndex`? :p How would you name it?